packages/libs/eda/src/lib/core/components/computations/plugins/correlationAssayMetadata.tsx

```diff
@@ -155,3 +161,66 @@ export function CorrelationAssayMetadataConfiguration(
     </ComputationStepContainer>
   );
 }
+
+const ASSAY_ENTITIES = [
+  'OBI_0002623',
+  'EUPATH_0000809',
+  'EUPATH_0000813',
+  'EUPATH_0000812',
+];
```

Comment:
Great question. I just made a [ticket](https://github.com/VEuPathDB/EdaNewIssues/issues/753) for it but i think it could use some more discussion. There wasn't originally a long-term plan because i didn't think that i would end up needing these assay ids at all until i got into the implementation.

To completely remove the hardcoded ids we'd need to add the following annotations:
1. To the study, add `hasMetagenomicData`. I think there's a similar `hasMapData` annotation?
2. To each entity, add `isAssay`. Or i suppose it could be something like `entityType = 'assay'`, if there are other entity types that might be useful to us.

Also open to other solutions!